<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadSettingWidget</class>
 <widget class="QWidget" name="DownloadSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选项设置</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>610</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>选项设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>592</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>29</y>
     <width>461</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#444444;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="first">
    <widget class="QLineEdit" name="downloadDirEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>48</y>
       <width>270</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="baseSetDownloadWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>基本设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadDirButton">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>47</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>更改目录</string>
     </property>
    </widget>
    <widget class="QLabel" name="downloadDirLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下载目录</string>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="startupDownloadWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>160</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>启动设置</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="startupCheckBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>185</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>开机启动</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="minimumRadioBox">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>120</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>最小化到系统托盘，不退出程序</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="quitRadioBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>120</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>直接退出程序</string>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="closeSetShowInfoWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>95</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>关闭主面板时</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="startupRunCheckBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>210</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>启动后自动开始未完成任务</string>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="slienceDownloadWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>255</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>静默下载</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="slienceRunCheckBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>280</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>启动静默下载</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="second">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="TTKThemeLineLabel" name="baseSetShotKeyWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>基本设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="item_lb1">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#3BA1E6;border-radius:2px</string>
     </property>
     <property name="text">
      <string>新建</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="DownloadGlobalHotKeyEdit" name="item_key1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <property name="text">
      <string>Ctrl+N</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item_lb2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>80</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#3BA1E6;border-radius:2px</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="DownloadGlobalHotKeyEdit" name="item_key2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <property name="text">
      <string>Ctrl+S</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="item_lb3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>110</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#3BA1E6;border-radius:2px</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="DownloadGlobalHotKeyEdit" name="item_key3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:white;</string>
     </property>
     <property name="text">
      <string>Ctrl+Q</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="globalHotkeyBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>150</y>
       <width>250</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>启用全局快捷键</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="third">
    <widget class="QComboBox" name="defaultDownloadModeBox">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>50</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="defaultDownloadModeLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>默认下载模式</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="taskmanagerChoiceWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>任务管理</string>
     </property>
    </widget>
    <widget class="QComboBox" name="downloadMaxCountBox">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>80</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="downloadMaxCountLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>同时下载的最大任务数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="downloadCountRangeLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>(1-10)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="downloadSpeedLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>215</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>KB/s</string>
     </property>
    </widget>
    <widget class="QLabel" name="downloadLimitSpeedLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>215</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="downloadLimitRadioBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>185</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>网速保护模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="uploadLimitSpeedLabel">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>215</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>上传</string>
     </property>
    </widget>
    <widget class="QLabel" name="uploadSpeedLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>215</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>KB/s</string>
     </property>
    </widget>
    <widget class="QComboBox" name="downloadLimitSpeedComboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>212</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="uploadLimitSpeedComboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>212</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="downloadFullRadioBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>160</y>
       <width>110</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下载优先模式</string>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="downloadModeWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>130</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>模式设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="four">
    <widget class="TTKThemeLineLabel" name="appearanceSettingWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>外观设定</string>
     </property>
    </widget>
    <widget class="QComboBox" name="effectLevelBox">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>50</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="effectLevelLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>特效等级</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="TTKThemeLineLabel" name="suspensionModeWidget">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>95</y>
       <width>415</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:13px;
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>悬浮窗设定</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="suspensionVisiableBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>125</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>悬浮窗显示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="suspensionShowPerBox">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>155</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>悬浮窗文字显示进度</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="functionArea" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>125</width>
     <height>425</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,200)</string>
   </property>
   <widget class="QLabel" name="baseSetWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
font-size:12px;
color:rgb(80,80,80);
font-weight:bold;</string>
    </property>
    <property name="text">
     <string>基本设置</string>
    </property>
   </widget>
   <widget class="DownloadFunctionTableWidget" name="normalFunTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>125</width>
      <height>125</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>410</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="functionAreaDisplay" native="true">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>29</y>
     <width>486</width>
     <height>425</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>functionAreaDisplay</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>functionArea</zorder>
  <zorder>confirmButton</zorder>
  <zorder>cancelButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTKThemeLineLabel</class>
   <extends>QLabel</extends>
   <header>ttkthemelinelabel.h</header>
  </customwidget>
  <customwidget>
   <class>DownloadFunctionTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">downloadsettingwidget.h</header>
  </customwidget>
  <customwidget>
   <class>DownloadGlobalHotKeyEdit</class>
   <extends>QLineEdit</extends>
   <header>downloadglobalhotkeyedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
